### ISSUE TITLE
tslib-common bad signature (Bugzilla Bug 408)

### ISSUE THREAD

**Issue by tpgxyz** [OpenMandrivaAssociation/test2#408]
This issue was created automatically with bugzilla2github
# Bugzilla Bug 408
Date: 2013-12-14 16:01:01 +0000
From: luca &lt;<agoiza@gmail.com>&gt;
To:   OpenMandriva QA &lt;<bugs@openmandriva.org>&gt;
CC:   @benbullard79, @itchka, siriustheking@yahoo.com, @tpgxyz

Duplicates:   #344
Depends on:   #344
Last updated: 2014-02-25 09:24:33 +0000

## Comment 2690
Date: 2013-12-14 16:01:01 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

The following package has bad signature:
/var/cache/urpmi/rpms/tslib-common-1.0-7-omv2013.0.i586.rpm: Invalid Key ID (OK (DSA/SHA1, sab 20 apr 2013 05:16:58 CEST, Key ID d3da3b1b04ae7750))
Do you want to continue installation ? (y/N)

## Comment 2692
Date: 2013-12-14 19:51:41 +0000
From: @benbullard79

Confirming.

## Comment 2710
Date: 2013-12-16 17:20:51 +0000
From: Gus Ballan &lt;<siriustheking@yahoo.com>&gt;

I get a lot of Invalid key ID/Missing key. I don't inform them anymore, as they keep appearing all the time. It seems that:

- The problem is much more big than some isolated packages, and/or
- The QA is being too slow to solve this issue.

For example, almost every time I add something to E17 window manager, the Invalid key error appears.

The list of affected packages is getting bigger: see bug #344 (filled Nov. 22nd, still unsolved) and bug #284 (filled on Nov. 8th, still unsolved too).

Inconsistency: bug #284 marked as trivial (probably I did it, because system works even after this error), this bug marked as critical, bug #344 not classified.

## Comment 2712
Date: 2013-12-17 01:03:43 +0000
From: @benbullard79

I'm trying to find someone who can teach me how to get packages correctly signed. This is a big issue with QA-Team. 

FWIW: As far as us being slow, well we are. We are all volunteers. We do what we can when we can. And I get frustrated with how long it takes to get some issues addressed myself. That's part of the down side of being a community distro. At one point I thought we'd get more support from Mandriva SA but that didn't happen. Everyone who does anything with OpenMandriva currently is a part time unpaid volunteer.

## Comment 2725
Date: 2013-12-17 20:32:37 +0000
From: Gus Ballan &lt;<siriustheking@yahoo.com>&gt;

(In reply to comment #3)
>
> FWIW: As far as us being slow, well we are. We are all volunteers. We do
> what we can when we can. And I get frustrated with how long it takes to get
> some issues addressed myself. That's part of the down side of being a
> community distro. At one point I thought we'd get more support from Mandriva
> SA but that didn't happen. Everyone who does anything with OpenMandriva
> currently is a part time unpaid volunteer.

Yes, I understand you. I know that "more hands" are needed. Even so, I think that some minor things should be solved quicker. For example, it seems that there is some troubles with the latest nVidia driver, like in http://forums.openmandriva.org/en/discussion/comment/1624 or existing bugs like bug #403. Bug #403 was filled by Davide Garatti, @luca confirmed it, and @MauRice confirmed it too, so I consider that bug #403 is an actual "running" bug. In bug #403 @luca proposed to remove the last update. It's not a big thing. Just remove 3 or 4 files from a repository, up to the time of the problem being solved. But updates are still there (confirmed yesterday night), so more users are prone to be victims of the bug. This is the "slowness" I'm complaining about.

And in the case of the "bigger" troubles (internationalization, grub2, hp printers, invalid keys), I don't see much "motion" in the ML either, so it's not clear when the problems will be solved.

All before is pretty off-topic. Going back to this bug, at least inform us if we must consider the invalid keys problem as something critical, as a major problem, or as something with what we can keep on walking without major concerns. I think that they're "non fatal" errors. But something is true: we cannot live much more time having invalid keys all the time. It's a serious security problem.

## Comment 2726
Date: 2013-12-17 23:16:42 +0000
From: @itchka

You will all be pleased to know that bad signature issue has been been handed over to QA and within the next few days we will start resigning the problem packages, at least those that have had a problem reported.
For information the problem is a bug within ABF the build system on which the packages are built. Although ABF has the necessary keys it does not always sign the packages. The problem is unfortunately not within our control we have filed a bug with the maintainers of ABF and we hope that it will be resolved soon.
This means that we will still need you to report bad-signatures because we cannot do anything with the packages that are now in the repository; we can however deal with updated packages because we can catch them at the testing stage.

## Comment 2828
Date: 2013-12-23 15:10:24 +0000
From: @tpgxyz

fixed -in -8 release

## Comment 2831
Date: 2013-12-23 20:37:54 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

fixed for me.

thanks.

## Comment 2834
Date: 2013-12-24 12:36:07 +0000
From: Gus Ballan &lt;<siriustheking@yahoo.com>&gt;

It's not fixed at all.

Today, a new update for tslib-common, version 1.0-8, as arrived.

The invalid key error still appears.

## Comment 2846
Date: 2013-12-25 23:46:18 +0000
From: @tpgxyz

lower the importance and severity

## Comment 3303
Date: 2014-02-25 09:24:33 +0000
From: @tpgxyz

We are working on this. Duplicate of bug #344

*** This bug has been marked as a duplicate of bug #344 ***
